src/multialign/MultiAlignerResult.hpp

```diff
@@ -0,0 +1,15 @@
+//
+// Created by chris on 11/20/23.
+//
+
+#pragma once
+#include <unordered_map>
+#include "BasicClasses/Forward.hpp"
+
+namespace MultiAlign
+{
+    struct MultiAlignerResult{
+        double score;
+        std::unordered_map<LigandID, PoseID> poseIDsByLigandID;
+    };
```

Comment:
std::vector<MultiAlign::Ligand>& ligands

We need this vector here for the OutputWriter. To access molecules by ID.